File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_wsservermessage_pong.rs


pub fn convert__wsservermessage__pong__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::ws_server_message::Pong) -> crate::victron_controller::dashboard::ws_server_message::Pong {
    crate::victron_controller::dashboard::ws_server_message::Pong {
        body: serde_json::from_value(serde_json::to_value(&from.body).unwrap()).unwrap(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_wsservermessage_pong.ts
// @ts-nocheck
import {Pong as v0_2_0_Pong} from './WsServerMessage'
import {Pong as v0_1_0_Pong} from '../v0_1_0/WsServerMessage'

export function convert__ws_server_message__pong__from__0_1_0(from: v0_1_0_Pong): v0_2_0_Pong {
    return new v0_2_0_Pong (
        JSON.parse(JSON.stringify(from.body))
    )
}